libs/application/templates/aosh/seminars/src/fields/Participants/index.tsx

```
@@ -0,0 +1,390 @@
+            maxSize={FILE_SIZE_LIMIT}
+          />
+        )}
+      />
+
+      {csvInputError.length > 0 &&
+        csvInputError.map((csvError: CSVError) => {
+          const messageString = `${formatMessage(
+            participantMessages.labels.csvErrorLabel,
+          )} ${csvError.items.join(', ')} - ${formatMessage(csvError.error)}`
+          return <AlertMessage type="error" message={messageString} />
+        })}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)